src/main/java/com/opinta/temp/StartHsqlDbManager.java

```diff
@@ -10,6 +10,6 @@
 public class StartHsqlDbManager {
     @PostConstruct
     public void startDBManager() {
-        DatabaseManagerSwing.main(new String[]{"--url", "jdbc:hsqldb:mem:testdb", "--user", "sa", "--password", "sa"});
+        //DatabaseManagerSwing.main(new String[]{"--url", "jdbc:hsqldb:mem:testdb", "--user", "sa", "--password", "sa"});
```

Comment:
This code is responsible only for invocation of irritative HSQLDB window every time you run in-memory tests. There are no point to retain all this bean anyway as it does nothing except throwing window. We've discussed it.